blocks/newsletter-modal/newsletter-modal.js

```diff
@@ -0,0 +1,101 @@
+import { getLibs } from '../../scripts/utils.js';
+
+function closeModal() {
+  const qModals = document.querySelectorAll('.dialog-modal');
+  if (qModals?.length) {
+    qModals.forEach((modal) => {
+      if (modal.nextElementSibling?.classList.contains('modal-curtain')) {
+        modal.nextElementSibling.remove();
+      }
+      modal.remove();
+    });
+    window.history.pushState('', document.title, `${window.location.pathname}${window.location.search}`);
+  }
+}
```

Comment:
Why recreate the closeModal?

Reply:
This is used for the close button after the signup. Is there already a functionality for the use case?
<img width="449" alt="Screen Shot 2023-03-28 at 10 44 53 AM" src="https://user-images.githubusercontent.com/25802201/228277116-55a70bfe-ae34-4740-b79b-40f93138f8af.png">

Reply:
Also, it's strange to have two close buttons but if that's that the designs say